### ISSUE TITLE
March Team Meeting 2019

### ISSUE THREAD

**Issue by Zsailer** [jupyterhub/team-compass#122]
Hello everyone, 

The March meeting will be held on the *third* Thursday this next month.

That's **Thursday, March 21th, 2019** at 6pm Zurich time (or your [timezone](http://arewemeetingyet.com/Zurich/2019-03-21/18:00/Jupyterhub%2FBinder%20team%20meeting)).

* You can find the agenda [here](https://hackmd.io/u2ghJJUCRWK-zRidCFid_Q?both).
* And we'll meet on Zoom (video conferencing) [here](https://calpoly.zoom.us/my/jupyter)
* 🎥 🎬 we will record the meeting for those who can't make it.

These calls are open to **anyone**, so please join us! We love seeing new members on the call.

**Comment by betatim** [comment]
What do you think of encouraging/nudging/forcing people to write down their thoughts/input on an agenda item before the meeting starts?

I like being able to see an agenda item ahead of time, think about it a bit and write down what I want to say. Compared to making it up on the fly.

It would also be a way for those who don't feel comfortable to speak up in the meeting (e.g. a fast paced verbal discussion in English is hard to follow for them) to give input.

**Comment by Zsailer** [comment]
Hi everyone, 

A few of the core Jupyterhub/Binder team members are at the in-person Jupyter team meeting. The schedule for the meeting is busier than we (or at least, I) originally thought. I don't think we'll be able to break away for the video call. 

So I propose we move our video call/meeting to next Thursday. This follows our usual third-Thursday-of-the-month. Any objections?

**Comment by Zsailer** [comment]
We've moved our meeting to next week. That's **Thursday, March 21st, 2019** at 6pm Zurich time. (I've updated the post above.)

This is your one week reminder! 😃

**Comment by betatim** [comment]
Make sure to check when this meeting is in your local timezone. We will stick with it being fixed to 6pm Zurich time, which is now a different time than usual in the USA (as the US is already on daylight saving but Europe isn't yet).

**Comment by Zsailer** [comment]
That's a good point. Thanks, @betatim. 

I'm updated the "timezone" link above. e

**Comment by lheagy** [comment]
Hi All, an idea for meeting structure... I really like the way [Pangeo hosts meetings](http://pangeo.io/meeting-notes.html#id1). The meeting starts with "Quick Updates" from everyone in attendance (~1min each) to tell everyone what you have been working on, raise any questions or propose topics for longer discussion. Then there are quick shout-outs for updates that people should know about, or things to celebrate. Then the meeting moves to longer discussion items (30 - 40 min total) that were documented before the meeting starts. This gives everyone a chance to speak, lets people bring up urgent items, while still tackling the bigger-picture topics that were planned beforehand. It seems to be a fairly efficient meeting structure, so might be worth giving a try?

**Comment by choldgraf** [comment]
Another note that there's some conversations happening about team meeting structure here as well:

https://discourse.jupyter.org/t/two-great-articles-on-running-more-effective-inclusive-meetings/448/4

I like the structure that @lheagy proposes - I'm also a fan of a (relatively strict) set the agenda at least X hours before the meeting structure.

Finally, it would be great if we could find a way to handle the "raise your hand to add yourself to the queue of talking" method in a remote meeting.

**Comment by betatim** [comment]
> it would be great if we could find a way to handle the "raise your hand to add yourself to the queue of talking" method in a remote meeting.

How about we use the hackmd we already have open. One person is in charge of calling on people and keeping track of where we are in the list. To get on the list you write your name below the point we are discussing. A bit like [this](https://hackmd.io/ifmjp26pRPK72IJr1Ygatg?edit)

It could turn into total chaos as five people try to add themselves at the same time but maybe that isn't so common? Might be worth a first try.

Otherwise we can always build a notebook with WebRTC where you can press a button to get yourself added to the list which is displayed in some other notebook ... j/k.

**Comment by choldgraf** [comment]
@betatim I like the idea of using hackmd...another option is to have a single list that people's names get deleted from after they've spoken, rather than one list per topic.

**Comment by leportella** [comment]
I won't be able to attend this one, but hopefully see you all in the next :)

**Comment by Zsailer** [comment]
@leportella You'll be missed!

**Comment by Zsailer** [comment]
@lheagy Thanks for sharing Pangeo's meeting structure! That structure looks great. I think the main difference is the *one minute quick updates*. We should definitely add this. My only hesitation is that someone might want to attend the call *just to listen*. It might make them uncomfortable to put them on the spot?

**Comment by Zsailer** [comment]
Also, this is your reminder to add your topics to the agenda if you haven't already! The agenda closes in ~24 hours.

**Comment by betatim** [comment]
> It might make them uncomfortable to put them on the spot?

Saying "pass" is an option for those who don't want to talk or have nothing to say. Let's try how that works?

> It might be too much to facilitate the call and manage this queue.

Good point. I'd be happy to help out or we could rotate the queue manager position by topic depending on who wants to be active during each topic.

**Comment by Zsailer** [comment]
Thank you to everyone who joined the call today. It was fun! 

We recorded the call if you couldn't be there. I'll share the link to the video as soon as I learn how to do that. 😉
